app/src/org/commcare/views/widgets/MediaWidget.java

```diff
@@ -237,21 +235,12 @@ public void setBinaryData(Object binaryuri) {
             }
         }
 
-        if (newMedia.exists()) {
-            // Add the copy to the content provider
```

Comment:
PS: Though, this change will only prevent crashes and won't change the earlier behaviour.